Marker/Classes/Markdown/MarkdownString.swift

```diff
@@ -44,11 +44,11 @@ internal func attributedStringFromParsedString(parsedString: String, tags: [Mark
 // Source: http://stackoverflow.com/questions/25138339/nsrange-to-rangestring-index/30404532#30404532
 private extension String {
     
-    func NSRangeFromRange(range: Range<String.Index>) -> NSRange {
-        let from = String.UTF16View.Index(range.startIndex, within: utf16)
-        let to = String.UTF16View.Index(range.endIndex, within: utf16)
+    func NSRangeFromRange(_ range: Range<String.Index>) -> NSRange {
```

Comment:
is `NSRangeFromRange` a custom function name or are you overriding something? Otherwise I'll rename `range(from range:)`